modules/rosa-hcp-sts-creating-a-cluster-ext-auth-provider-cli.adoc

```diff
@@ -0,0 +1,150 @@
+// Module included in the following assemblies:
+//
+// * rosa_hcp/rosa-hcp-sts-creating-a-cluster-ext-auth.adoc
+:_mod-docs-content-type: PROCEDURE
+[id="rosa-hcp-sts-creating-a-cluster-external-auth-provider-cli_{context}"]
+= Creating an external authentication provider
+:source-highlighter: pygments
+:pygments-style: emacs
+:icons: font
+ 
+After you have created a {hcp-title} cluster with the enabled option for external authentication providers, you must create a provider using the ROSA CLI.
```

Comment:
```suggestion
After you have created a {hcp-title} cluster with external authentication providers, you must create a provider using the ROSA CLI.
```
Is "the enabled option" necessary here? I feel like its just extra and can be removed. Disregard this comment if it is.